### ISSUE TITLE
Problema con latitud y longitud

### ISSUE THREAD

**Issue by AdrianMBarrera** [taroull/tourism#1]
Hola @Julia92 ,

hemos estado probando los últimos cambios que subiste. Está genial que ya devuelva la latitud y la longitud, con el ejemplo de playas funciona perfecto.

Sin embargo, en casos como este, devuelve error: http://www.taro.ull.es:8983/tourism/v1/accommodation?name=

![captura de pantalla 2015-05-27 a las 9 39 23](https://cloud.githubusercontent.com/assets/6286419/7832122/749e05c6-0454-11e5-90da-f1961721e605.png)

Es porque algún valor (la latitud o la longitud) devuelve vacío. Creo que simplemente sería poner una condición en el Find del HTOService. Algo parecido a lo que hay con el código postal.

Cualquier cosa, nos avisas por aquí o por correo.

Gracias y saludos!

**Comment by Julia92** [comment]
Hola, muchísimas gracias por avisarme, creo que ya está bien, ayer me despisté de añadirlo en el momento y casualmente todas las pruebas que hice después tenían latitud y longitud. Desde que termine con el RDF de los recursos volcánicos, intento añadir los datos que faltan en estos RDF. Igualmente, cualquier cosa que necesiten o se den cuenta estaré por aquí.

De nuevo gracias y saludos :)

**Comment by AdrianMBarrera** [comment]
Pues lo probamos ya y no dió error. Pero hemos detectado otras cosillas:
- Se están repitiendo algunos resultados. La consulta de los restaurantes, por ejemplo, devuelve más de 2000 resultados, y debería de devolver muchos menos. Por ejemplo, si hacemos la siguiente consulta: http://www.taro.ull.es:8983/tourism/v1/gastro?name=oasis

```
[  
   {  
      postalCode:"38670",
      latitude:28.122086,
      longitude:-16.724539,
      uri:"http://turismo-de-tenerife.org/resource/restauracion/Oasis",
      name:"Oasis"
   },
   {  
      postalCode:"38250",
      latitude:28.122086,
      longitude:-16.724539,
      uri:"http://turismo-de-tenerife.org/resource/restauracion/Oasis",
      name:"Oasis"
   },
   {  
      postalCode:"38670",
      latitude:28.122086,
      longitude:-16.345158,
      uri:"http://turismo-de-tenerife.org/resource/restauracion/Oasis",
      name:"Oasis"
   },
   {  
      postalCode:"38250",
      latitude:28.122086,
      longitude:-16.345158,
      uri:"http://turismo-de-tenerife.org/resource/restauracion/Oasis",
      name:"Oasis"
   },
   {  
      postalCode:"38670",
      latitude:28.554295,
      longitude:-16.724539,
      uri:"http://turismo-de-tenerife.org/resource/restauracion/Oasis",
      name:"Oasis"
   },
   {  
      postalCode:"38250",
      latitude:28.554295,
      longitude:-16.724539,
      uri:"http://turismo-de-tenerife.org/resource/restauracion/Oasis",
      name:"Oasis"
   },
   {  
      postalCode:"38670",
      latitude:28.554295,
      longitude:-16.345158,
      uri:"http://turismo-de-tenerife.org/resource/restauracion/Oasis",
      name:"Oasis"
   },
   {  
      postalCode:"38250",
      latitude:28.554295,
      longitude:-16.345158,
      uri:"http://turismo-de-tenerife.org/resource/restauracion/Oasis",
      name:"Oasis"
   }
]
```

   No sé si los resultados están bien o si debería devolver sólo uno. Ya nos comentas.
- Otra cosa que nos hemos dado cuenta es que las coordenadas están un poco mal, pero eso ya no depende de nosotros, ya que son los datos de Turismo de Tenerife los que están mal.

¡Saludos!

**Comment by Julia92** [comment]
Sí, me di cuenta antes pero la verdad es que no se si está así por algo del año pasado, voy a seguir investigando a ver si encuentro la solución, porque no entiendo por qué se van intercambiando los códigos postales y la latitud y la longitud, y en otros casos se duplica varias veces lo mismo.
![casa pepe](https://cloud.githubusercontent.com/assets/2444922/7836889/d0d37bc4-047b-11e5-868d-61e2b5abf6f5.PNG)

También encontré recursos que luego no encuentro dentro de http://www.opendatacanarias.es/datos/dataset/tdt-restauracion como el Benchijigua Express, pero si están en el RDF que está dentro de la API. Es que en este caso me di cuenta, porque el Benchijigua Express es el barco de Fred Olsen.
![benchijigua express](https://cloud.githubusercontent.com/assets/2444922/7836937/0f72b8b8-047c-11e5-85f4-5dd5b1ad09d1.PNG)

¿Podría cambiar el RDF que está dentro por los que están en http://www.opendatacanarias.es/datos/dataset/tdt-restauracion para ver si así se soluciona?

Saludos :)

**Comment by AdrianMBarrera** [comment]
El problema es que esos RDFs son viejos, si quieres, Luis compartió en el Drive los RDFs más actualizados (con las URLs sin tildes, espacios, etc...).

Respecto a lo del Benchijigua, al parecer lo han eliminado y no está en las nuevas versiones de los RDFs.

De todas formas, estos RDFs no están actualizados a día de hoy (ni los que están en la web ni los del Drive), así que lo miramos y te comentamos.

Sé que Laura estaba intentando hacer las consultas directamente al punto SPARQL de Open Data Canarias, pero no sé cómo lo lleva. Puedes hablar con ella, a ver qué prefiere hacer.

Saludos!

**Comment by Julia92** [comment]
De acuerdo, muchísimas gracias!
Saludos.
